admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -711,7 +711,6 @@
   }
 
   #product-images-dropzone.dropzone {
-    display: flex;
```

Comment:
I'm not sure if this one was needed, @NeOMakinG I think you're the one who integrated this component right? Do you remember?

Reply:
If you remove the flex, you have to remove every flex property above

If it works without it, then we don't need it! My guess is that it's also used for the image-selected window, isn't it? It might generate an issue there

Reply:
hmm this is how it looks and i don't see what could be the issue. Tried to add other changes that where suggested but I think just removing flex looks the best.
![image](https://user-images.githubusercontent.com/51944825/190639491-7eb3e841-1c29-4d7a-9954-8edddd4b9bfc.png)